fenix/app/src/test/java/org/mozilla/fenix/home/recenttabs/controller/RecentTabControllerTest.kt

```diff
@@ -88,13 +86,11 @@ class RecentTabControllerTest {
             navController.navigate(R.id.browserFragment)
         }
         assertNotNull(RecentTabs.recentTabOpened.testGetValue())
-        assertNull(RecentTabs.inProgressMediaTabOpened.testGetValue())
     }
 
     @Test
     fun handleRecentTabClickedForMediaTab() {
```

Comment:
❓ Do we even need this test now that this telemetry is not being recorded. It's similar to the test above

Reply:
I think we should keep this in case someone thought this was a mistake in the future.

Reply:
good call, this would fail if the condition was added back in the code 👍